chainer/sequential.py

```diff
@@ -381,6 +381,11 @@ def copy(self, mode='share'):
                 ret.append(copy.copy(layer))
         return ret
 
+    def copyparams(self, sequential, copy_persistent=True):
```

Comment:
>`sequential`

How about not changing the parameter name from the base method? I don't think it's a good practice.
By the way, can we assume the source link is of `Sequential`? (There's no such mention of the assumption in the documentation).

Reply:
OK, I’ll fix the name of this argument.

> can we assume the source link is of Sequential?
Ah, I guess it’s better to raise an error when the given source link is not a Sequential object?
And at least I need to add a description about it to the documentation.

Reply:
Thanks.